### ISSUE TITLE
Doing the same. Did you go any further? 

### ISSUE THREAD

**Issue by Biker1bob** [dabonetn/ZimodemESP8266Leds#2]
Hi, I am using a ESP 12f module and a FULL TTL breakout DB9 - I was having issues with the LEDs myself.
Do you feel like sharing any more info?  Not a pro with Github, I see your schematic, and I assume there is some firmware in here somewhere. The last version I have from bo is 3.6.1 but not sure its his last.. Do you feel up to making an updated version?  Or I can flash yours I guess I am thinking the 74244 etc is what I am missing.

James

**Comment by dabonetn** [comment]
The latest version from Bo ZImmer has led support built in now.
And yes, the 74244 is required for this to work.

This guy made a premade solution for sale.

https://tempestfpga.com/retromodem/

**Comment by Biker1bob** [comment]
I am builder, just not a designer. I can build this no issues, just not sure on the design. You had a WIFI LED - Does the latest version have output to a pin for that?  I am using a ESP-12f nodemcu board. I need to figureout - and if you could help that would be fantastic - what pin out is the wifi led on my board. I found a built up version of the firmware the first time. I will examine Bo Zimmers github myself as well. Thanks - I will paste my circuit up here if its ok when I am done. I am modifying your drawing.

**Comment by Biker1bob** [comment]
Here is my design - seem right to you? 560ohm on the led resistors seem ok?
![Basic Schematic](https://user-images.githubusercontent.com/33105915/202249770-a74036f5-ef9a-40c3-865b-ff4e892cd72b.jpg)

**Comment by dabonetn** [comment]
The design looks basically like a copy of mine, so it should work, as for the pin assignment, look in the zimodem.ino file and all the pins are defined. 
You then need to lookup the Arduino pint numbers for your esp board to make sure they match.

You also have to enable them with a option.

define SUPPORT_LED_PINS

As for the value of your resistors, that's up to you. I had to play around with the values to get a approximate match to the old modem brightness, and this is totally dependent on the LEDs you are using.

**Comment by Biker1bob** [comment]
yes I have done all that. Which I am in Arduino 1.8.13 I open zimodem.ino add the line to the top(which I did) Then I can compile it right there correct? do I need to add a library or anything? just compile the sketch and upload correct?

**Comment by dabonetn** [comment]
Give it a try, it's been a long time since I played with this.

**Comment by Biker1bob** [comment]
Got it all working well in my Avatex 1200hc. Only issue I have now is trying to get the settings correct in order to host a bbs.  Not enough discussion going on about Zimodem itself.. not sure how to figure out whats not working. I also have several wimodem232 from Jim Drew - great product, but commercial firmware.. so I cant use it for my DIY modem.
